pex/tools/commands/venv.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import
+
+import errno
+import os
+import shutil
+import zipfile
+from argparse import ArgumentParser, Namespace
+from textwrap import dedent
+
+from pex import pex_builder, pex_warnings
+from pex.common import chmod_plus_x, safe_mkdir
+from pex.environment import PEXEnvironment
+from pex.pex import PEX
+from pex.tools.command import Command, Error, Ok, Result
+from pex.tools.commands.virtualenv import PipUnavailableError, Virtualenv
+from pex.tracer import TRACER
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Tuple
+
+
+def _copytree(
+    src,  # type: str
+    dst,  # type: str
+    exclude=(),  # type: Tuple[str, ...]
+    collisions_ok=False,  # type: bool
+):
```

Comment:
A comment explaining "why not `shutil.copytree`" might be good.

Reply:
Agreed - done.